parts/k8s/manifests/kubernetesmaster-kube-controller-manager.yaml

```diff
@@ -19,6 +19,7 @@ spec:
       env:
       - name: AZURE_ENVIRONMENT_FILEPATH
         value: "/etc/kubernetes/azurestackcloud.json"
+      <gosdkloglevel>
```

Comment:
Why can't we simply include the configuration in here explicitly? We're already in a if azure stack scenario here.

cc @jadarsie

Reply:
Correct, it should be fine to wrap it in a  `{{- if IsAzureStackCloud}}` block.

Reply:
Made suggested change, please take a look.